doc/guide/usage.rst

```diff
@@ -82,6 +82,19 @@ Execute the system :var:`command` with the given :var:`arguments`, with a
 proper `$PATH` (see :ref:`cask path`) and `$EMACSLOADPATH` (see :ref:`cask
 load-path`).
 
+.. _cask emacs:
+
+cask emacs
+----------
+
+.. program:: cask emacs
+
+::
+   cask [GLOBAL-OPTIONS] emacs [ARGUMENTS ...]
+
+Execute `emacs` with the given :var:`arguments`, with the appropriate
+environmment (see :ref:`cask exec`). The Emacs executable is that which cask
+would normally run in.
```

Comment:
Please clarify the documentation.  Explain what the "Emacs executable which cask would normally run in" actually means, and document that the executable comes from `$EMACS` or `$CASK_EMACS`.